src/App.js

```diff
@@ -1,13 +1,25 @@
 import React from "react";
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
 import logo from "./logo.svg";
-import Header from "./components/Header"
+import Header from "./components/Header";
 import Projects from "./pages/Projects";
+import Welcome from "./pages/Welcome";
 
 function App() {
   return (
```

Comment:
Lets add a comment todo here to for fixing the route link

Reply:
Done :)